web/src/components/modals/AddInstructionModal.tsx

```diff
@@ -30,25 +30,28 @@ export default function AddInstructionModal() {
   }
 
   return (
-    <DefaultModalLayout
-      icon={SvgAddLines}
-      title="Set Project Instructions"
-      description="Instruct specific behaviors, focus, tones, or formats for the response in this project."
-      mini
-    >
-      <div className="bg-background-tint-01 p-4">
-        <InputTextarea
-          value={instructionText}
-          onChange={(event) => setInstructionText(event.target.value)}
-          placeholder="Think step by step and show reasoning for complex problems. Use specific examples."
+    <Modal open={modal.isOpen} onOpenChange={modal.toggle}>
+      <Modal.Content mini>
+        <Modal.Header
+          icon={SvgAddLines}
+          title="Set Project Instructions"
+          description="Instruct specific behaviors, focus, tones, or formats for the response in this project."
+          onClose={() => modal.toggle(false)}
         />
-      </div>
-      <div className="flex flex-row justify-end gap-2 p-4">
-        <Button secondary onClick={() => modal.toggle(false)}>
-          Cancel
-        </Button>
-        <Button onClick={handleSubmit}>Save Instructions</Button>
-      </div>
-    </DefaultModalLayout>
+        <Modal.Body className="bg-background-tint-01 p-4">
```

Comment:
Maybe we should have `p-4` as the default padding on the `Modal.Body` component?

Also, there are some modals which have a `tint-01` background, and some that have a `tint-00` background. Since this is such a common pattern, let's create new flags on the `Modal.Body` component. It could look something like:

```tsx
<Modal.Body light> { ... } </Modal.Body>
<Modal.Body dark> { ... } </Modal.Body>
```